controllers/storagecluster/platform_detection.go

```diff
@@ -6,18 +6,12 @@ import (
 	"sync"
 
 	configv1 "github.com/openshift/api/config/v1"
-	"github.com/red-hat-storage/ocs-operator/controllers/util"
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	//IBMCloud COS[Cloud Object Storage] secret name
-	ibmCloudCosSecretName = "ibm-cloud-cos-creds"
-
 	//IBMCloudPlatform with COS Secret
 	IBMCloudCosPlatformType configv1.PlatformType = "IBMCloudCosPlatform"
```

Comment:
let's remove this constant as well wherever it is used as we are not setting this anymore

Reply:
Done